Makefile

```diff
@@ -130,7 +130,7 @@ bashate: ## Run bashate.
 	hack/bashate.sh
 
 .PHONY: ci-job
-ci-job: common-deps-update generate fmt vet golangci-lint unittest shellcheck bashate update-bindata bundle-check imager-unittest
```

Comment:
You should also be able to drop the go-bindata import from the dependencymagnet file and clean up the vendor folder:
https://github.com/openshift-kni/lifecycle-agent/blob/main/pkg/dependencymagnet/doc.go#L10

As well, remove the other go-bindata components added by the original commit:
https://github.com/openshift-kni/lifecycle-agent/commit/dc7c5671a5f4ff4274e0b8d78b79a976c75bf8f4#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

From that, it looks like you may also be able to drop the build-machinery-go import from dependencymagnet.

cc: @jc-rh

Reply:
thanks, updated